### DISCUSSION TITLE
Corbin Crutchley

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#191] [Announcements]
- https://twitter.com/crutchcorn
- https://unicorn-utterances.com/unicorns/crutchcorn
- https://unicorn-utterances.com/collections/framework-field-guide
- https://github.com/houseform/houseform
- https://stars.github.com/profiles/crutchcorn/

## Intro

Andrew: Hello and welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

## Themes

- Community Building
- Framework wars
- OSS Contribution

## Background/OSS

- [x] What's your approach to open source?
- [x] What project do you contribute to?
- [x] You have some pretty cool libraries. You're cli testing library is super cool. What's your favorite?
- [x] Why did you build another react form library?
- [x] You just backed a GitHub star, congrats, what does that mean in practice?

### Book/Frameworks

- [x] What is the book you're writing about? 
- [x] First book coming up to release, what's for the other two? What does the first cover?
- [x] How do the three frameworks differ/are similar?
- [x] Did you're journey learning all the frameworks on a deep level lead you to a new favorite?
- [x] What new chapter do you have planned?
- [x] For the internals chapter you plan to build a toy version of each framework. Have you done any of this work already? Which one are you excited to build? How far will you take it?
- [x] What do you think the best framework for a beginner is?
- [x] You're writing your own framework: https://crutchcorn.github.io/the-fun-framework/ Why? What's that about? What have you learned?

### Future

- [x] What's your spiciest web dev take?
- [x] What's the future of these frameworks? Future of webdev?

**Comment by hipstersmoothie** [comment]
https://github.com/silvia-odwyer/photon
https://github.com/oedotme/generouted

**Comment by crutchcorn** [comment]
# Topics

- First book coming up to release, what's for the other two? What does the first cover?
- How do the three frameworks differ/are similar?
- What's the future of these frameworks? Future of webdev?
- You're writing your own framework: https://crutchcorn.github.io/the-fun-framework/ Why? What's that about? What have you learned?

# Corbin's Tooltips

- [GPD Win Max 2](https://www.gpd.hk/gpdwinmax2) (my personal laptop)
- [Typora](https://typora.io/)
- [Git Fork](https://git-fork.com/)
- [Bomb Pops](https://bombpop.com/) 

> Feel free to cut any of these - the "Bomb Pops" is mostly an in-joke with my community as a shoddy attempt to get them to sponsor me (Bombpops, if you're reading this - call me 🤙🤙)

**Comment by just-be-dev** [comment]
https://build.mmm.page/
https://markwhen.com/
